deb/build/debian/jenkins.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Jenkins Continuous Integration Server
+After=network.target
+
+[Service]
+Type=simple
+User=jenkins
+Group=jenkins
+SyslogIdentifier=jenkins
+UMask='0002'
+LimitNOFILE='8192'
+
+Environment=JENKINS_HOME=/var/lib/jenkins
+ExecStart=/usr/bin/java \
```

Comment:
I definitely agree. The reason I have not reused the `deb/build/debian/jenkins.default` provided by the package is that it is meant to be interpreted by a shell when systemd only supports `KEY=Value`.

I guess an other file should be provided, I am not sure what dh_systemd recommends for that though.